config/config.example.yml

```diff
@@ -258,3 +258,9 @@ mediaViewer:
 info:
   enableEndUserAgreement: true
   enablePrivacyStatement: true
+# Home Page
+homePage:
+	# The number of item showing in recent submission components
+  recentSubmissionsRpp: 5
+	# Sort record of recent submission
+  recentSubmissionsSortField: 'dc.date.accessioned'
```

Comment:
We have another pull request #1803 which is adding the same sort of "homePage" configuration, and ideally we'd like these two PRs to have configurations that look similar.  

Could we restructure this configuration to look like this?
```
homePage:
  recentSubmissions:
    pageSize: 5
    sortField: 'dc.date.accessioned'

```

This would allow both this PRs to use the same "homePage" section of the configuration and work similarly.  I also think this will be easier for users to understand, as the "Rpp" meaning is not clear.